lib/OctokitAction.ts

```diff
@@ -34,8 +34,8 @@ export abstract class OctokitAction extends Action {
     return this.graphqlWithAuth(query);
   }
 
-  protected getInput(name: string): string {
-    return core.getInput(name);
+  protected getInput(name: string, required: boolean = false): string {
+    return core.getInput(name, { required });
```

Comment:
Why this change? Required parameters are defined in the `action.yml`, and if anything is optional, it should have a default value there. And then, inside TS, we don't need to care.

Reply:
How does it work?
I made the change because I did not update the workflow after we moved from having only `JIRA_TOKEN` to having both `JIRA_USER` and `JIRA_TOKEN`.
If I don't set those inputs, which are required, I am having an error during the action execution:
```
401 (Unauthorized): You do not have permission to create issues in this project.
```

I am expecting to have an issue before and a failed workflow then.
It seems that even with the required field in TS, it does not work.

Reply:
I reverted this change

Reply:
If you don't send parameter that action.yml requires, it should fail quickly. But I didn't test it.
If you send useless parameters that are not expected by action.yml, you'll get a warning.